examples/push/main.go

```diff
@@ -1,22 +1,20 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"gopkg.in/src-d/go-git.v4"
 	. "gopkg.in/src-d/go-git.v4/examples"
 )
 
 func main() {
-	CheckArgs("<repo path>")
-	repoPath := os.Args[1]
+	CheckArgs("<repository-path>")
```

Comment:
Here we can add a comment with an example path. After it must be a path to a .git folder. Now I think it is not necessary.